signer-wasm/README.md

```diff
@@ -1,3 +1,12 @@
 # Filecoin WASM Library
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+## Structure
+
+* `ledger-filecoin-js` submodule which refers to https://github.com/Zondax/ledger-filecoin-js
+* `src` rust bindings for javascript
+* `pkg` ready to distribute wasm module
```

Comment:
I am confused with this.. I assume this is because of the submodule?
I saw in another place that /pkg was removed

Reply:
Yes it is a submodule with the wasm generated code which is ready to be used.

I don't think I have removed /pkg